### PR TITLE
🐛 Fix more broken links

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
Discovered intended way to publish docs/README.md.

## Summary
This PR replaces one relative link to non-doc contents and recognizes the intended way to publish docs/README.md.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @clubanderson 
@ezrasilvera

File: GOVERNANCE.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The KubeStellar and its leadership embrace the following values:
 
 KubeStellar Maintainers have write access to the [project GitHub repository](https://github.com/kubestellar/kubestellar).
 They can merge their own patches or patches from others. The current maintainers
-can be found as top-level approvers in [OWNERS](./OWNERS).  Maintainers collectively 
+can be found as top-level approvers in [OWNERS](https://github.com/kubestellar/kubestellar/blob/main/OWNERS).  Maintainers collectively
```
Comment (ezrasilvera) [inline GOVERNANCE.md:50]: Please note this remove the dependency on version.

Comment (MikeSpreitzer) [reply GOVERNANCE.md:50]: That is intentional. I think that for this file, it is always the current version that is relevant.

**Pull request merged**